sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.sql.catalyst.expressions
 
 import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
```

Comment:
Nit: these should be grouped as `java.text.{DecimalFormat, DecimalFormatSymbols}`